src/main/java/org/example/onlinebookstore/dto/user/ResponseUserDto.java

```diff
@@ -0,0 +1,4 @@
+package org.example.onlinebookstore.dto.user;
+
+public record ResponseUserDto(Long id, String email) {
```

Comment:
```suggestion
public record UserResponseDto(Long id, String email) {
```
Let’s  provide all user information in response (exclude password)